war/src/main/less/modules/manage-jenkins.less

```diff
@@ -1,152 +1,175 @@
 /* ----------- Manage Jenkins page ----------- */
 
-.manage-messages {
-    margin-bottom: 20px;
-}
-
 .manage-messages a,
 .manage-messages a:visited {
-    color: inherit !important;
-    text-decoration: none;
+  color: inherit !important;
+  text-decoration: none;
 }
 
 .manage-messages a:hover,
 .manage-messages a:focus {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages dl:first-child {
-    margin-top: 0;
+  margin-top: 0;
 }
 
 .manage-messages dl dt:first-child {
-    margin-top: 0;
+  margin-top: 0;
 }
 
 .manage-messages dl dt {
-    margin-top: 10px;
-    font-weight: normal;
+  margin-top: 10px;
+  font-weight: normal;
 }
 
 .manage-messages dl dd {
-    margin-left: 15px;
+  margin-left: 15px;
 }
 
 .manage-messages dl:last-child {
-    margin-bottom: 0;
+  margin-bottom: 0;
 }
 
 .manage-messages dl dt::after {
-    content: ": ";
+  content: ": ";
+}
+
+.manage-messages {
+  .alert:last-of-type {
+    margin-bottom: 30px;
+  }
 }
 
 .manage-messages .alert a {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages .alert a:hover {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 .manage-messages .alert form {
-    position: relative;
-    float: right;
-    margin: -6px 0 0 0 !important;
+  position: relative;
+  float: right;
+  margin: -6px 0 0 0 !important;
 }
 
 .manage-messages .alert form span {
-    margin: 0 0 0 4px !important;
+  margin: 0 0 0 4px !important;
 }
 
-.manage-option {
-    border: 1px solid transparent;
-    margin-bottom: 4px;
+.jenkins-settings {
+  &__section-title {
+    margin: 0 0 calc(2rem) 0;
+    font-size: 0.8rem;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    opacity: 0.75;
+  }
+
+  &__section {
+    border-top: 2px solid var(--panel-border-color);
+    padding: calc(2rem) 0;
+    max-width: 1800px;
 
-    &:hover,
-    &:focus-within {
-      border: 1px solid var(--manage-option-border-color);
+    &:first-of-type {
+      border-top: none;
+      padding-top: 0;
     }
-}
 
-.manage-option dl {
-    padding: 0;
-    margin: 0 0 0 62px;
-    min-height: 48px;
-}
+    .manage-page__row {
```

Comment:
This is sort of what I meant, tracking the `.jenkins-settings__section .mange-page__row` to this file would be very quite hard doing file search.